Add content from community doc PRs (745, 764, 773, 779) (#72)
rancher/harvester-product-docs: versions/v1.6/modules/en/pages/upgrades/troubleshooting.adoc

image::upgrade/upgradelog_archive.png[The upgrade log archive contains all the logs generated by the upgrade-related Pods]

> After the upgrade ended, {harvester-product-name} stops collecting the upgrade logs to avoid occupying the disk space. In addition, you can click the *Dismiss it* button to purge the upgrade logs.

Reviewer: Very minor suggestion, but the phrasing made me pause. Ignore if you don't like it.

The upgrade process generates (rather than collect?) logs. After it finishes, how can it 'stop collecting upgrade logs'. It's already finished so can't 'stop collecting' more logs.

Maybe I'm missing something.

```suggestion
After the upgrade ends, if the upgrade logs are not required, you can click the *Dismiss it* button to purge the upgrade logs.
```

Author: This is legacy content that I have not had time to clean up. I'd like to review and update the section as a whole instead of just this one part. There are many more language errors like this.
